### PR TITLE
docs: feature release blog for gh-ost(beta)

### PR DESCRIPTION
**Pull request by candy2255**

### CODE REVIEW

**Review by tianzhou — CHANGES_REQUESTED** [review]
Also please change the filename

File: content/blog/Introducing-gh-ost-based-online-schema-migration-for-mysql [hunk 1]
```diff
@@ -0,0 +1,72 @@
+---
+title: “Introducing gh-ost Based Online Schema Migration for MySQLL”
+author: Candy
+published_at: 2022/08/09 16:37:17
+feature_image: /static/blog/Introducing-gh-ost-based-online-schema-migration-for-mysql/cover.webp
```
[the remark this answers is not in the document]
Comment (candy2255) [reply content/blog/Introducing-gh-ost-based-online-schema-migration-for-mysql]: OK

File: content/blog/gh-ost-based-online-schema-migration-for-mysql.md [hunk 2]
```diff
@@ -0,0 +1,72 @@
+---
+title: “gh-ost Based Online Schema Migration for MySQL”
```
Comment (tianzhou) [inline content/blog/gh-ost-based-online-schema-migration-for-mysql.md:2]:
```suggestion
title: gh-ost Based Online Schema Migration for MySQL
```

File: content/blog/gh-ost-based-online-schema-migration-for-mysql.md [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+By adopting these tools, teams can reduce schema migration downtime from days and hours to seconds. On the other hand, operating these tools still requires a lot of expertise. And their usage is limited to the seasoned DBAs, while Developers still need to ask their DBA peers to perform online schema changes on their behalf.
+
+Since Bytebase is a database tool for the DevOps teams, we want to hand over the powerful online schema change capability to the Developers.
+
+In this blog post, we will introduce you to the beta release of Bytebase online schema migration solution based on gh-ost. Bytebase solution provides a step-by-step interface with continuous progress updates to oversee the entire migration process.
+
+## Why We Choose gh-ost
+
+All existing online schema change solutions share a similar workflow:
```
Comment (tianzhou) [inline content/blog/gh-ost-based-online-schema-migration-for-mysql.md:26]: nice illustration

File: content/blog/gh-ost-based-online-schema-migration-for-mysql.md [hunk 4]
```diff
@@ -0,0 +1,72 @@
...
+Bytebase provides two types of schema migration: regular migration and gh-ost. You can operate it by creating an issue in three steps. Let’s demonstrate how to use gh-ost to alter schema.
+
+The screenshot below shows the table “sbtest1” information before applying schema migration. We can see that the table has 4 columns and 694 MB data.
+![overview](/static/blog/gh-ost-based-online-schema-migration-for-mysql/overview.webp)
+
+**Step 1**. Create a gh-ost issue
+1. Click **Alter Schema** on the database details page. 
+![step_1_1](/static/blog/gh-ost-based-online-schema-migration-for-mysql/step_1_1.webp)
+
+2. Select **Online migration**and click **Next.**
```
Comment (tianzhou) [inline content/blog/gh-ost-based-online-schema-migration-for-mysql.md:48]:
```suggestion
2. Select **Online migration** and click **Next.**
```

File: content/blog/gh-ost-based-online-schema-migration-for-mysql.md [hunk 5]
```diff
@@ -0,0 +1,72 @@
...
+![step_1_2](/static/blog/gh-ost-based-online-schema-migration-for-mysql/step_1_2.webp)
+
+3. Enter SQL statements for the task “Sync data”, and click **Create** after selecting a reviewer to approve the issue.
+![step_1_3](/static/blog/gh-ost-based-online-schema-migration-for-mysql/step_1_3.webp)
+
+**Step 2**. Execute the task “Sync data”.
+The task “Sync data” runs after the designated reviewer clicks **Approve**. See the screenshot below, it shows that the progress of “Sync data” is 59%. 
+![step_3](/static/blog/gh-ost-based-online-schema-migration-for-mysql/step_3.webp)
```
Comment (tianzhou) [inline content/blog/gh-ost-based-online-schema-migration-for-mysql.md:56]: image is missing

**Review by tianzhou — CHANGES_REQUESTED** [review]

**Review by tianzhou — DISMISSED** [review]
LGTM. Nice work

**Review by tianzhou — APPROVED** [review]
LGTM

**Pull request merged**